pytest_doctestplus/plugin.py

```diff
@@ -88,11 +89,13 @@ def pytest_addoption(parser):
                   "Default format for docs. "
                   "This is no longer recommended, use --doctest-glob instead.")
 
+    parser.addini("text_file_format", "changing default format for docs")
```

Comment:
Remove this? (it looks like it is already defined above)

Reply:
oh, i probably messed up rebase